core/src/main/scala/org/bitcoins/core/wallet/utxo/UTXOSpendingInfo.scala

```diff
@@ -353,6 +365,19 @@ object BitcoinUTXOSpendingInfoFull {
     }
   }
 
+  def apply(
+      spendingInfoSingle: UTXOSpendingInfoSingle,
+      signers: Vector[Sign]): BitcoinUTXOSpendingInfoFull =
```

Comment:
Should probably require that `signers.contains(spendingInfoSingle.signer)` so that people don't accidentally omit it